blueskyacademy/bluesky-nextjs
# add new design for about page

## Description
- Add new design for about page and teacher page 

## Why
- For purpose: user can view about page and teacher page


## How to QA
- User can view the about page and teacher page
- User can see the animation when scrolling component into view

## Screenshots
![Screen Shot 2021-12-22 at 15 06 28](https://user-images.githubusercontent.com/74655110/147057766-ac2122a8-f90a-47fb-a99f-46dbf6988e13.png)
![Screen Shot 2021-12-22 at 15 06 43](https://user-images.githubusercontent.com/74655110/147057774-4e3e6bce-4a3e-487d-8d24-0a63db4dfd14.png)
